select.go

```diff
@@ -238,8 +342,10 @@ func (e *ESql) convertValExpr(expr *sqlparser.Expr) (dsl string, err error) {
 	case *sqlparser.SQLVal:
 		dsl = sqlparser.String(*expr)
 		dsl = strings.Trim(dsl, `'`)
+	case sqlparser.ValTuple:
+		dsl = sqlparser.String(*expr)
 	default:
-		err = errors.New("esql: not supported rhs expression")
+		err = fmt.Errorf("esql: not supported rhs expression %T", *expr)
```

Comment:
I think you should consider change all functions input `expr *sqlparser.Expr` to just `expr sqlparser.Expr` since Expr is already interface. This will save you tons of efforts dereference everywhere. (I don't think elasticsql make a good example on using this point style)

Reply:
modified